app/pb_kits/playbook/pb_card/_card.html.erb

```diff
@@ -1,7 +1,9 @@
 <%= content_tag(:div,
     id: object.id,
     data: object.data,
-    class: object.classname(object.kit_class),
+    class: object.classname,
     aria: object.aria) do %>
-  <%= object.yield(context: self) %>
+  <% if object.block %>
+    <%= pb_rails("card/card_body", props: { padding: object.padding}) { capture(&object.block) } %>
```

Comment:
I think we could make `Block` a proptype so it can be passed either as a prop `block: object.block` or as a block, so we avoid reimplementing the capture. Have you tried?

Reply:
No. I'll give it shot.

Reply:
@xjunior - Conceptually, I get this. But implementation wise, I'm stabbing in the dark. Care to add this change yourself, or perhaps we could pair on Monday?